### ISSUE TITLE
Possible feature: button to remove frame from data set

### ISSUE THREAD

**Issue by WillJStone** [Cartucho/OpenLabeling#60]
When labeling frames of a video it is a common occurrence to have some frames with nothing in them that you want to label. Would be nice to have a button that removes a selected frame from the data set (including its .xml/.txt file). Any interest in this?

**Comment by Cartucho** [comment]
Yes.

Do you think when we remove the selected frames we should also split the video into two? I am thinking about the case where it could crash the tracker.

**Comment by WillJStone** [comment]
Yes that makes sense to me. Since we know there is nothing in that frame for the tracker to track it would make sense to explicitly tell it to stop when it gets to that point.
